src/raygun.rum/core-web-vitals.js

```diff
@@ -6,12 +6,13 @@
  * raygun4js
  * https://github.com/MindscapeHQ/raygun4js
  *
- * Copyright (c) 2021 MindscapeHQ
+ * Copyright (c) 2023 MindscapeHQ
```

Comment:
👀

Reply:
I have bumped this as well as updated all other mindscape copyright notes to use the current year